charts/rasa-x/templates/rasa-deployments.yaml

```diff
@@ -108,6 +108,8 @@ spec:
           value: "{{ .rasaEnvironment }}"
         - name: "RASA_MODEL_SERVER"
           value: "http://{{ include "rasa-x.host" $ }}:{{ $.Values.rasax.port }}/api/projects/default/models/tags/{{ .modelTag }}"
+        - name: "RASA_DUCKLING_HTTP_URL"
+          value: "http://{{ include "duckling.host" . }}:{{ $.Values.duckling.port }}"
```

Comment:
```suggestion
          value: "http://{{ include "duckling.host" $ }}:{{ $.Values.duckling.port }}"
```

Reply:
🤦‍♀ thank you

Reply:
no facepalm please. That's a speciality of our chart and the for loop we are using in this file.